File: Linguagem C/1009.c
#include <stdio.h>

int main()
{
   char N[10];
   double S,V,T;

   scanf("%s",&N);
   scanf("%lf",&S);
   scanf("%lf",&V);

   T = S+(V*0.15);

   printf("TOTAL = R$ %.2f\n",T);

    return 0;
}

File: Linguagem GO/1009.go
package main
 
import (
    "fmt"
)
 
func main() {
    
    var nomeVendedor string
    var salarioFixo, totalVendas, salarioLiquido float64
    
    fmt.Scanf("%s",&nomeVendedor)
    fmt.Scanf("%f",&salarioFixo)
    fmt.Scanf("%f",&totalVendas)
    
    salarioLiquido = salarioFixo + totalVendas * 0.15
    fmt.Printf("TOTAL = R$ %.2f\n",salarioLiquido)
    
}
